### DISCUSSION TITLE
Question on "Robot collision Representation"

### DISCUSSION THREAD

**Discussion by zhangsanfeng86** [NVlabs/curobo#26] [Software Q&A] [answered]
Hi,

After "generate spheres" ,  the "save“ button not activate.
Could you give me some help?

![image](https://github.com/NVlabs/curobo/assets/73547638/8d206e20-ec80-4bdd-b280-415042e5356b)

**Comment by balakumar-s** [comment, marked as answer]
In the `Output File` section, did you click the folder button and type a file name to save the file? It looks like you haven't set a file name. Once you set a filename, the save button should get activated.

**Reply by zhangsanfeng86** [reply]
![image](https://github.com/NVlabs/curobo/assets/73547638/987cc6e2-0233-42c7-935d-379fc459e866)
I set a file name , but it not activated.

**Reply by zhangsanfeng86** [reply]
![image](https://github.com/NVlabs/curobo/assets/73547638/d27e4670-cd40-424c-9e58-eb5b7982af8d)
I also get a "Invalid path" warning

**Comment by zhangsanfeng86** [comment]
Sorry,    file type is "yaml".

**Reply by balakumar-s** [reply]
Did everything work?
